techniques/general/G217.html

```diff
@@ -2,137 +2,114 @@
+               また、コンテンツ制作者は、修飾キーの追加に加えて、割り当てられた文字キーの変更を利用者に許可することもできる。
+               ショートカットをオフにするか再割り当てするメカニズムを提供することで、より多くの利用者がアプリケーションと正常に対話できるようになる。
             </p>
             <div class="note">
-               <div role="heading" class="note-title marker" aria-level="3">Note</div>
-               <p>If the keyboard shortcut is only active when a particular user interface component
-                  has focus, then the author does not need to provide an override mechanism. For example,
-                  in a <code>select</code> element, it is expected behaviour that pressing a letter key will reposition to the
-                  next item within the select element that begins with that character. This shortcut
-                  is only triggered when the select element has focus, and so it does not interfere
-                  with a user's ability to interact with the rest of the page.
+               <div role="heading" class="note-title marker" aria-level="3">注記</div>
+               <p>特定のユーザインターフェースコンポーネントにフォーカスがあるときにのみキーボードショートカットが有効になっている場合、コンテンツ制作者は上書きするメカニズムを提供する必要はない。
+                  たとえば、<code>select</code> 要素では、文字キーを押すと、その文字で始まる select 要素内の次の項目に移動することが予想される。
+                  このショートカットは、select 要素にフォーカスがある場合にのみトリガされるため、利用者がページの残りの部分と対話する機能を妨げることはない。
                </p>
             </div>
-            <p>An initial challenge for testers is identifying if author-created shortcuts exist,
-               and then determining if existing shortcuts are triggered by character keys without
-               modifiers. Where testers have access to the develop team, the existence and nature
-               of shortcut keys can often be determined simply by asking involved designers and developers.
-               Where reliable information from the authors is not available, the presence of shortcuts
-               can possibly be identified by checking code (for example, in javascript, the presence
-               of <code>keydown</code>, <code>keyup</code> and <code>keypress</code> listeners). Another means of identifying shortcuts is to review documentation. Where
-               none of these strategies provide information on the presence of keyboard shortcuts,
-               manual tests will need to be completed to search for the existence of character key
-               shortcuts. Review the test procedure in <a href="https://w3c.github.io/wcag/techniques/failures/F99">F99: Failure of Success Criterion 2.1.4 due to implementing character key shortcuts that
-                  cannot be turned off or remapped</a> for guidance.
+            <p>テスタに​​とっての最初の課題は、コンテンツ制作者が作成したショートカットが存在するかどうかを特定し、既存のショートカットが修飾子なしの文字キーによってトリガされるかどうかを判断することである。
+               テスタが開発チームにアクセスできる場合、ショートカットキーの存在と性質は、多くの場合、関係するデザイナや開発者に尋ねることで判断できる。
+               コンテンツ制作者からの信頼できる情報が利用できない場合、ショートカットの存在はコードをチェックすることでおそらく識別できる (たとえば、javascriptで、<code>keydown</code>、<code>keyup</code>、および <code>keypress</code> listenerの存在)。
```

Comment:
（長音をどうするかはさておき、）listenerはイベントリスナーのことでしょう。

```suggestion
               コンテンツ制作者からの信頼できる情報が利用できない場合、ショートカットの存在はコードをチェックすることでおそらく識別できる (例えば、javascript で、<code>keydown</code>、<code>keyup</code>、及び <code>keypress</code> リスナーの存在)。
```